### PR TITLE
v0.2.22

### PR DESCRIPTION
**Pull request by dualcnhq**
#### Release tag for:
- more-facets
- loading tile styling: adjusted color, gradient direction, border radius

### CODE REVIEW

**Review by latonv — APPROVED** [review]
👍

File: src/collection-facets/facets-template.ts [hunk 1]
```diff
@@ -85,6 +92,7 @@ export class FacetsTemplate extends LitElement {
   private getFacetsTemplate(facetGroup: FacetGroup): TemplateResult {
     let facetsBucket = facetGroup?.buckets as FacetBucket[];
 
+    if (!facetsBucket) return html`${nothing}`;
```
Comment (iisa) [inline src/collection-facets/facets-template.ts:95]: Q: why is this file also committed?

Comment (dualcnhq) [reply src/collection-facets/facets-template.ts:95]: there's a failing test related to it, i added this part as fix

Comment (dualcnhq) [reply src/collection-facets/facets-template.ts]: will fix the failing test on a different PR

**Pull request merged**